detection-rules/spam_abused_indeed.yml

```diff
@@ -0,0 +1,27 @@
+name: "Spam: Abused Indeed Application Emails"
+description: "Indeed application emails are being used to deliver spam by injecting an image into the application template and hiding the default Indeed content via inline CSS. This has been observed in the delivery of emails containing a lure themed of account/membership expiration of popular online services.  Instead of renewing their membership, the recipient signs up for a Consumer Savings Club which charges a monthly membership fee."
+type: "rule"
+severity: "low"
+source: | 
+  type.inbound
+  // from indeed
+  and strings.ends_with(headers.message_id, '-ApplicationEmail@indeedemail.com>')
+  and headers.auth_summary.dmarc.pass and headers.auth_summary.dmarc.details.from.domain == 'indeedemail.com'
+  
+  // with a hidden section
+  and not strings.like(body.html.raw,
+                   '*style="display:none;">___*',
+                   '*style="display: none;">______*',
+                   '*<div style="display: none; max-height: 0px; overflow: hidden;">&nbsp;*'
+  
+  )
+attack_types:
+  - "Spam"
+tactics_and_techniques:
+  - "Evasion"
+  - "Image as content"
+  - "Abused Service Provider"
```

Comment:
I know this is a draft so more of an FYI.

`Abused Service Provider` isn't an option today, we originally had an option for Infrastructure abuse, but it didn't make the cut.

Reply:
Ahhh, I assumed it was a bit free form.  I'll take a look and see if I can find a list.